test/e2e/e2e_test.go

```diff
@@ -461,7 +459,7 @@ func triggerForcedRotation(t *testing.T, client *kubernetes.Clientset, config *r
 
 	// Trigger a forced rotation by updating the operator config
 	// with a reason.
-	setUnsupportedServiceCAConfig(t, config, "42", customDuration)
+	forceUnsupportedServiceCAConfigRotation(t, config, secret, customDuration)
```

Comment:
(No action required) Not sure what this change is buying given this function is only called once in a test run. YAGNI and all.

Reply:
This is the “Fix failures on repeated e2e tests in the same cluster” commit. 

The current behavior is perfectly fine for a CI run; when developing the tests, repeatedly running them on a single cluster, this failure is distracting from any real failures or test bugs.